modules/mo_pytorch/setup.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
Is this file needed in this PR?

Reply:
Dmitry already created a separated PR for these mo_pytorch changes: https://github.com/openvinotoolkit/openvino_contrib/pull/18
I applied these changes in advance to pass some security scans. 
@dkurt would you like me to remove your changes from my PR?

Reply:
That's for SDL process checks (originally from https://github.com/openvinotoolkit/openvino_contrib/pull/18)

Reply:
I never mind to keep it in this PR. Changes are trivial

Reply:
Perhaps git rebase should be applied to keep patches and history clean.